scripts/Makefile

```diff
@@ -0,0 +1,97 @@
+
+${GEOCINT_WORK_DIRECTORY}/geocint/installation/runner_installer: ${GEOCINT_WORK_DIRECTORY}/geocint/installation/preparation_block_finished ## install dependencies of runner part 
+	bash ${GEOCINT_WORK_DIRECTORY}/geocint-runner/runner-install.sh || echo 'runner-install.sh returned an error, check logs for more infornation'
+	touch $@
+
+${GEOCINT_WORK_DIRECTORY}/geocint/installation/openstreetmap_installer: ${GEOCINT_WORK_DIRECTORY}/geocint/installation/runner_installer ## install dependencies of openstreetmap part
+	bash ${GEOCINT_WORK_DIRECTORY}/geocint-openstreetmap/openstreetmap-install.sh || echo 'openstreetmap-install.sh returned an error, check logs for more infornation'
+	touch $@
+
+${GEOCINT_WORK_DIRECTORY}/geocint/installation/h3_pg_installation: ${GEOCINT_WORK_DIRECTORY}/geocint/installation/openstreetmap_installer ## install h3_pg
+	sudo pgxn install h3
+	touch $@
+
+${GEOCINT_WORK_DIRECTORY}/geocint/installation/private_installer: ${GEOCINT_WORK_DIRECTORY}/geocint/installation/h3_pg_installation ## install dependencies of openstreetmap part
+	bash ${GEOCINT_WORK_DIRECTORY}/${CUSTOM_PART_FOLDER_NAME}/install.sh || echo 'install.sh returned an error, check logs for more infornation'
+	touch $@
+
+${GEOCINT_WORK_DIRECTORY}/geocint/installation/set_modes: ${GEOCINT_WORK_DIRECTORY}/geocint/installation/private_installer ## create public_html folder and set access
+	sudo usermod -aG adm ${USER}
+	sudo usermod -aG ${USER} www-data
+	sudo mkdir -p ${GEOCINT_WORK_DIRECTORY}/public_html
+	sudo chown ${USER}:${USER} ${GEOCINT_WORK_DIRECTORY}/public_html
+	sudo chmod 0750 ${GEOCINT_WORK_DIRECTORY}/public_html
+	sudo echo 'export PATH=$$PATH:/usr/local/pgsql/bin' >> ${HOME}/.bashrc
+	echo 'GEOCINT_WORK_DIRECTORY=${GEOCINT_WORK_DIRECTORY}' | sudo tee -a /etc/environment
+	touch $@
+
+${GEOCINT_WORK_DIRECTORY}/geocint/installation/installation_block_finished: ${GEOCINT_WORK_DIRECTORY}/geocint/installation/set_modes ## send message that installation step finished successfully
+	echo "Installation step finished successfully"
+	touch $@
+
+
+# ------------------------ DB configuration block ----------------------------
+
+${GEOCINT_WORK_DIRECTORY}/geocint/installation/create_user: ${GEOCINT_WORK_DIRECTORY}/geocint/installation/installation_block_finished ## create postgresql user
+	sudo -u postgres psql -c "CREATE ROLE ${PGUSER} WITH SUPERUSER LOGIN ENCRYPTED PASSWORD '${PGUSER}pass';"
+	touch $@
+	
+${GEOCINT_WORK_DIRECTORY}/geocint/installation/create_database: ${GEOCINT_WORK_DIRECTORY}/geocint/installation/create_user ## create postgresql db
+	sudo -u postgres psql -c "CREATE DATABASE ${PGDATABASE} WITH OWNER ${PGUSER};"
+	touch $@
+	
+${GEOCINT_WORK_DIRECTORY}/geocint/installation/create_extensions: ${GEOCINT_WORK_DIRECTORY}/geocint/installation/create_database ## create postgresql extensions
+	echo "Create PostgreSQL extensions:"
+	echo ${PG_EXTENSIONS_LIST}	
+	for i in $(PG_EXTENSIONS_LIST); do \
```

Comment:
it's hard to tell if extensions list should be configurable but it just symptom of bigger problem that geocint-runner currently doesn't add any new useful abstractions but limits limits configuration capabilities.